source/.ninja-usage

```diff
@@ -0,0 +1,42 @@
+\e[1m\e[33mUsage: \e[97mninja [\e[1;36mOPTIONS\e[97m] [\e[36mTARGETS...\e[97m]\e[0m
```

Comment:
I see these stray characters displayed as-is in my terminal - what are they?   Color markers?

Reply:
Yes, odd that your terminal does not support ansi! It's not like that's a very new standard 🤔 We can take it out. It's pretty though :)

Reply:
It looks like the problem is with the literal `\e` to mean the Escape char in the `.ninja-usage` text.  If I replace `\e` with the actual code for escape character  - ^[ - I get the correct output.

Reply:
This gives the right output on my system:

```python
w.rule('print-help', r"cat .ninja-usage | sed 's/\\e/^[/g'")
```

Reply:
I changed `\e` to `\033`; maybe that works.